examples/outsim-advanced-ts/.env

```diff
@@ -0,0 +1,4 @@
+DEBUG=node-insim-outsim-basic-ts
+HOST=127.0.0.1
+PORT=29999
+OUTSIMOPTS=0x1ff
```

Comment:
This `OUTSIMOPTS` env variable doesn't seem to be used in the example app. Could you add it in the `index.ts` file so it's used?

Reply:
good point - used env variable value in `index.ts`